core/src/main/scala/kafka/admin/AdminUtils.scala

```diff
@@ -608,7 +608,7 @@ object AdminUtils extends Logging with AdminUtilities {
   def fetchAllChildEntityConfigs(zkUtils: ZkUtils, rootEntityType: String, childEntityType: String): Map[String, Properties] = {
     def entityPaths(zkUtils: ZkUtils, rootPath: Option[String]): Seq[String] = {
       val root = rootPath match {
-        case Some(path) => rootEntityType + '/' + rootPath
+        case Some(path) => rootEntityType + '/' + path
```

Comment:
@ijuma Oops, will add a test. Would it be better if I created a PR with this change and tests? It will be good to apply the change to the 0.10.1 branch as well. Thank you for spotting it.